README.md

```diff
@@ -66,10 +66,12 @@ Configuration options can be specified on the command line such as
 `-DdependencyExcludes="io.github.mfoo:*"` or as part of the plugin
 configuration in `pom.xml`.
 
-| Option               | Description                                                 | Format                          |
-|----------------------|-------------------------------------------------------------|---------------------------------|
-| `dependencyExcludes` | Ignore certain dependencies                                 | `io.github.mfoo:*,org.apache:*` |
-| `maxLibYears`        | Cause the build to fail if dependencies are older than this | `4`                             |
```

Comment:
Should this `maxLibYears` setting interact with the years-without-new-update setting?

E.g, it might make sense to cause failures if either any dependency is more than X years behind the latest _or_ if it hasn't been updated in X years.

Reply:
We don't want the build to fail, we just need the information about how old the dependencies are and actually in one file.